vach/src/lib.rs

```diff
@@ -4,41 +4,31 @@
 #![deny(missing_docs)]
 
 /*!
-<p align="center">
-  <img src="https://raw.githubusercontent.com/zeskeertwee/virtfs-rs/main/media/logo.png" alt=".vach logo" width="180" height="180">
-</p>
-<p align=center> A simple archiving format, designed for storing assets in compact secure containers </p>
-<p align=center>
-  <a href="https://docs.rs/vach"><img alt="docs.rs" src="https://img.shields.io/docsrs/vach?style=flat-square"></a>
```

Comment:
Why remove all these badges?

Reply:
There are not needed in the API documentation on docs.rs, they are good for crates.io and the repo tho